cmd/server/assets/login/_loginscripts.html

```diff
@@ -38,3 +38,28 @@
   }
 </script>
 {{end}}
+
+{{define "pindiv"}}
+<div class="card shadow-sm" style="display:none;" id="pinDiv">
+  <div class="card-header">
+    SMS Pin
+    <button type="button" class="close" aria-label="Close" id="pinClose">
+      <span aria-hidden="true">&times;</span>
+    </button>
+  </div>
+  <div class="card-body">
+    <form id="pinForm" class="floating-form" action="/" method="POST">
+      <div class="form-label-group">
+        <input type="text" id="pin" name="pin" class="form-control" placeholder="Pin" autocomplete="one-time-code" required autofocus />
```

Comment:
We're using PIN and OTP interchangeably here - should standardize on one if possible.

Reply:
And, if we go with PIN, we should always capitalize it as "PIN"